src/status_im/contexts/wallet/send/routes/view.cljs

```diff
@@ -99,59 +123,97 @@
-                                  [quo/section-label
-                                   {:section         (i18n/label :t/from-label)
-                                    :container-style (style/section-label 0)}]
-                                  [quo/section-label
-                                   {:section         (i18n/label :t/to-label)
-                                    :container-style (style/section-label 64)}]]
-        :render-fn               (fn [item]
-                                   [route-item
-                                    {:amount       amount
-                                     :theme        theme
-                                     :fetch-routes fetch-routes
-                                     :status       (cond
-                                                     (= (:status item) :add)   :add
-                                                     loading-suggested-routes? :loading
-                                                     :else                     :default)
-                                     :from-network (if loading-suggested-routes?
-                                                     (utils/id->network item)
-                                                     (utils/id->network (get-in item [:from :chain-id])))
-                                     :to-network   (if loading-suggested-routes?
-                                                     (utils/id->network item)
-                                                     (utils/id->network (get-in item
-                                                                                [:to :chain-id])))}])}]
+        data                      (if loading-suggested-routes? affordable-networks routes)]
+    (if (or (and (not-empty affordable-networks) loading-suggested-routes?) (not-empty routes))
+      (let [data (if (not-empty disabled-from-networks)
+                   (add-disabled-networks data disabled-from-networks loading-suggested-routes?)
+                   data)
+            data (if (and (< (- (count data) (count disabled-from-networks)) 3)
+                          (pos? (count data)))
+                   (concat data [{:status :add}])
+                   data)]
+        [rn/flat-list
+         {:data                    data
+          :content-container-style style/routes-container
+          :header                  [rn/view {:style style/routes-header-container}
+                                    [quo/section-label
+                                     {:section         (i18n/label :t/from-label)
+                                      :container-style style/section-label-left}]
+                                    [quo/section-label
+                                     {:section         (i18n/label :t/to-label)
+                                      :container-style style/section-label-right}]]
+          :render-fn               (fn [item index]
```

Comment:
this render function is huge. Maybe it's better to separate to it's own `defn` within this file?

Reply:
Sure, will move it! Also, probably this will be refactored in the near future, I have some ideas to simplify it and being able to test different kind of routes, because currently we can test UI with that the suggested routes algorithm returns as response, but for example represented cases in the designs are difficult to achieve and now is also more difficult until bridges give more support to Sepolia.

Reply:
Nice! 👌